arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/ImcFragment.java

```diff
@@ -136,6 +140,30 @@ public Trace trace(Trace trace) {
       subFragment.trace(trace, hub.state.stamps());
     }
 
+    if (childFrame != null) {
+      trace.pMiscCcrsStamp(childFrame.revertStamp()).pMiscCcsrFlag(childFrame.selfReverts());
```

Comment:
Tracing of `CCRS` and `CCSR`. The child context revert information is accessed directly through the child call frame.